### DISCUSSION TITLE
Minimal tox.ini

### DISCUSSION THREAD

**Discussion by mtelka** [encode/httpcore#801] [Ideas]
Would you accept a PR with minimal `tox.ini`?  Something similar to [this one](https://github.com/typeddjango/pytest-mypy-plugins/pull/121).

Thanks.

**Comment by lovelydinosaur** [comment]
I appreciate `tox`, but it's not a tool that we use for our CI.

**Comment by mtelka** [comment]
Sure.  You do not need to use `tox` if you have `tox.ini` in the repo.  `tox.ini` would be just a convenience for downstream packagers.

Thank you.
